sdk/core/azure-core/azure/core/pipeline/transport/_requests_basic.py

```diff
@@ -225,14 +225,19 @@ def send(self, request, **kwargs): # type: ignore
         error = None # type: Optional[Union[ServiceRequestError, ServiceResponseError]]
 
         try:
+            timeout = kwargs.pop('connection_timeout', self.connection_config.timeout)
+            if not isinstance(timeout, tuple):
+                _LOGGER.warning("Tuple timeout setting is deprecated")
+                read_timeout = kwargs.pop('read_timeout', self.connection_config.read_timeout)
+                timeout = (timeout, read_timeout)
             response = self.session.request(  # type: ignore
                 request.method,
                 request.url,
                 headers=request.headers,
                 data=request.data,
                 files=request.files,
                 verify=kwargs.pop('connection_verify', self.connection_config.verify),
-                timeout=kwargs.pop('connection_timeout', self.connection_config.timeout),
+                timeout=timeout,
```

Comment:
I see this in logs:
`[WARNING _requests_basic]: Tuple timeout setting is deprecated`